dist/components/tabs/_tabs.scss

```diff
@@ -0,0 +1,117 @@
+// Tabs
+// ====
+//
+// Tabs component with tabs controls and tabs body.
+// Horizontal controls tabs items placed in one line.
+
+
+// Configurable Variables
+// ---
+
+$tabs__item-size: 150px !default;
+$tabs__padding: 10px 5px !default;
+$tabs__body-space: 20px !default;
+$tabs__active-background: hotpink;
```

Comment:
Looks like $tabs__item-size functions as a minimum width for tabs as they get added. A value in em units might be best as a base for the component. The exact value is probably best determined by guestimating based on expected content; the default value may not matter too much.